cmd/server/main.go

```diff
@@ -519,6 +535,13 @@ func (p *Proxy) runAgentServer(o *ProxyRunOptions, server *server.ProxyServer) e
 func (p *Proxy) runAdminServer(o *ProxyRunOptions, server *server.ProxyServer) error {
 	muxHandler := http.NewServeMux()
 	muxHandler.Handle("/metrics", promhttp.Handler())
+	if o.enableProfiling {
+		muxHandler.HandleFunc("/debug/pprof", pprof.Index)
```

Comment:
Not a blocker but I noticed that the links generated on the pprof index page when you hit /debug/pprof are for /debug/X not for /debug/pprof/X. So you get 404's if you click any of the links. The links are fine if you hit /debug/pprof/. You might consider using a redirect on "/debug/pprof" to send them to the good url.

func indexRedirect(w http.ResponseWriter, r *http.Request) {
	http.Redirect(w, r, "/debug/pprof/", http.StatusMovedPermanently)
}

Reply:
Done.  PTAL. Thanks.